aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java

```diff
@@ -579,6 +579,25 @@ public synchronized void terminate() throws DataStoreException {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void clear(@NonNull Action onComplete,
+                      @NonNull Consumer<DataStoreException> onError) {
+        try {
+            terminate();
+            sqliteStorageHelper.deleteDatabaseFromDisk();
+            onComplete.call();
+        } catch (Exception exception) {
```

Comment:
Let's try to make this exception handling more narrow. Which of these calls are actually expected to throw, and what types of exceptions will they throw?

Can you do:
```
try {
    justTheThingsThatThrow();
} catch (OnePossibleCaseException | AnotherCaseException failureToClear) {
    // wrap in DataStoreException and re-throw
}
```

Reply:
This method has been changes. Take a look at the updated logic and let me know.